### DISCUSSION TITLE
Addition of Bassetlaw District Council (United Kingdom)

### DISCUSSION THREAD

**Discussion by Graymattor** [mampfes/hacs_waste_collection_schedule#2211] [General]
Hi developers, 

Would it be possible to add Bassetlaw District Council (United Kingdom) to the Supported Services?

https://www.bassetlaw.gov.uk/bins-recycling-and-waste/bins-for-recycling-and-waste/bin-collection-days/

Thank you for the great work and regular updating the feature.

**Comment by 5ila5** [comment]
Please fill out [this](https://github.com/mampfes/hacs_waste_collection_schedule/issues/new?assignees=&labels=source+request&projects=&template=SOURCE-REQUEST.yml&title=%5BSource+Request%5D%3A+)

But it looks kinda hard, and they only show the weekday anyway, so you could just use the static source, or am I missing something where I can see a list of dates?
